plugins/jest/readme.md

```diff
@@ -0,0 +1,44 @@
+# @dotcom-tool-kit/jest
+
+A plugin to run [Jest](https://jestjs.io/) tests. This plugin uses [jest-cli](https://www.npmjs.com/package/jest-cli) to run the tests. You can configure this plugin by specifying the path to the Jest config path.
+
+## Installation
+
+With Tool Kit [already set up](https://github.com/financial-times/dotcom-tool-kit#installing-and-using-tool-kit), install this plugin as a dev dependency:
+
+```sh
+npm install --save-dev @dotcom-tool-kit/jest
+```
+
+And add it to your repo's `.toolkitrc.yml`:
+
+```yaml
+plugins:
+    - '@dotcom-tool-kit/jest'
+```
+
+## Options
+
+| Key | Description | Default value |
+|-|-|-|
+| `configPath` | Path to the [Jest config file](https://jestjs.io/docs/27.x/configuration) | N/A |
+
+## Tasks
+
+| Task | Description | Preconfigured hook |
+|-|-|-|
+| `JestLocal` | runs `jest` to execute tests | `test:local` |
+| `JestCI` | runs `jest` to execute tests in the CI with the `--ci` [option](https://jestjs.io/docs/cli#--ci) | `test:ci` |
+
+## To note
```

Comment:
**nitpick**: I think a heading like 'Tips' makes more sense but you can keep 'To note' if you prefer!
```suggestion
## Tips
```